plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -3311,7 +3359,14 @@ public void registerCoreMembers() {
         // @Author mcmonkey
         //
         // @Description
-        // TODO: Document Command Details
+        // Creates, loads , pastes and saves schemactics. Schematics are files containing info about
+        // blocks and the order of those blocks.
+        //
+        // Denizen offers a number of tools to manipulate and work with schematics.
+        // Schematics can be rotated, flipped, pasted with no air, or pasted with a delay.
+        // The "noair" option skips air blocks in the pasted schematics- this means those air blocks will not replace
+        // any blocks in the target location.
+        // The "delayed" option

```

Comment:
uh

Reply:
This entire change set ... what? You spelled schematic wrong on the first line, have all sorts of formatting issues, and just cut off at the end. What.